sdk/appconfiguration/app-configuration/test/public/utils/testHelpers.ts

```diff
@@ -197,3 +200,323 @@ export async function assertThrowsAbortError(
     }
   }
 }
+
+export function toWebResourceLike(request: PipelineRequest): WebResourceLike {
```

Comment:
do we need full copies of all this compat code as test helpers? what are we validating?

Reply:
I was lazy. Removed everything and simplified.
Tests are happy, I'm happy.